nemo_aligner/algorithms/supervised.py

```diff
@@ -133,6 +143,19 @@ def train_single_step(self, batch):
 
         return loss_mean, trainer_metrics | metrics
 
+    @torch.no_grad()
+    def run_generation(self, batch):
+        context_tokens, context_lengths = batch["contexts"], batch["context_lengths"]
```

Comment:
I think we should standardize the naming and keep it consistent.

can you use "text" and "length" on the batch here as well? https://github.com/NVIDIA/NeMo-Aligner/blob/09b3dccaf4aebfb515e5d531ee2cae4416c67ff3/nemo_aligner/models/nlp/gpt/megatron_gpt_ppo_actor.py#L279C42-L279C46